### PR TITLE
Hardfork update process

### PR DESCRIPTION
**Pull request by ygao76**
### What changed?
Consensus update process
- 

---

<sub>**Reminder:** If content (docs, blogs, pages) is moved or renamed, please ensure there are no broken links.</sub>

@NullSoldier @iron-fish/engineering

### CODE REVIEW

File: content/documentation/concensus_update_process.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Consensus Update Process
```
Comment (NullSoldier) [inline content/documentation/concensus_update_process.mdx:2]: I think this page is unused.

File: content/documentation/concensus-update-process.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+title: Consensus Update Process
+description: Consensus Update Process | Iron Fish Blockchain
+---
+
+To maintain consensus, all full nodes validate blocks using the same consensus rules. However, sometimes the consensus rules are changed to introduce new features or prevent network abuse. A hard fork will be performed when the new consensus rules are introduced. Hard fork creates
+a permanent divergence in the block chain, where non-upgraded nodes can’t validate blocks created by upgraded nodes that follow newer consensus rules.
+
+### Requirement for Hard fork
+1. Hard Fork FIP, including the full list of changes in the consensus
+2. New randomness for the circuit if trust setup is required for the hard fork
```
Comment (NullSoldier) [inline content/documentation/concensus-update-process.mdx:11]: This is not related to hard forks, this is related to circuit changes.

**Review by NullSoldier — APPROVED** [review]

File: content/documentation/concensus-update-process.mdx (hunk 2 above)
Comment (ygao76) [reply content/documentation/concensus-update-process.mdx:11]: I will rephrase it to `New randomness for the circuit if hard fork has any circuit change`, what do you think?
